usb/core.py

```diff
@@ -669,11 +669,18 @@ def __getitem__(self, index):
         """
         return Interface(self.device, index[0], index[1], self.index)
 
+    def _get_power_multiplier(self):
+        if self.device.speed is not None:
+            power_multiplier = _lu.MAX_POWER_UNITS_USB_SUPERSPEED if self.device.speed == 3 else _lu.MAX_POWER_UNITS_USB2p0
```

Comment:
```suggestion
            power_multiplier = _lu.MAX_POWER_UNITS_USB_SUPERSPEED if self.device.speed >= 4 else _lu.MAX_POWER_UNITS_USB2p0
```

---

`libusb_supported_speed` constants also form a bitmap, and `LIBUSB_SUPER_SPEED_OPERATION` has value 4, not 3.[^1]

And while some future SuperDuperSpeed mode will likely change the bMaxPower multiplier again (to some value we can't predict), it might be a bit better to (mis)handle that future case as SuperSpeed, instead of <= HighSpeed.

Thanks again!

[^1]: https://github.com/libusb/libusb/blob/cc498ded18fb2c6e4506c546d0351c4ae91ef2cc/libusb/libusb.h#L498

Reply:
Thanks for the suggestion - I've committed it. Is there anything else that you'd like me to change before it's pulled?